MovieApp/MovieApp/App/Features/Movies/Components/MovieCollectionViewCell/MovieCollectionViewCell.swift

```diff
@@ -14,6 +14,8 @@ class MovieCollectionViewCell: UICollectionViewCell {
     var ellipseImageView: UIImageView!
     var heartImageView: UIImageView!
     
+    var identifier: Int!
+    
```

Comment:
why do you need this?

Reply:
It enables me to differentiate id for each cell I tapp on.

Reply:
I understand. Forgot the primitive collectionView